tests/conftest.py

```diff
@@ -3,7 +3,8 @@
 from __future__ import absolute_import, division, print_function
 
 import os
-
+import sys
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```

Comment:
Am curious about this line. What is this for?

Reply:
I had to add that in for the tests to run locally in my dev environment like they used to do previously. (Is there a nicer way of doing this?) Without this line it's unable to find the `ispyb` package since it's not "pip installed", it just exists as a source code directory.